src/pages/index.tsx

```diff
@@ -1,3 +1,26 @@
+import Head from 'next/head';
+
 export default function Home() {
-  return <div>Hello, this is TaoTalk </div>;
+  return (
+    <>
+      <Head>
+        <title>TaoTalk</title>
+        <meta name="viewport" content="width=device-width, initial-scale=1" />
+        <meta name="description" content="TaoTalk" />
+        <link rel="icon" href="/favicon.ico" />
```

Comment:
Move this tag into _app file. This tag will always be the same with every single page.